hrui/css/hrui.css

```diff
@@ -5,3 +5,33 @@
 #crm-main-content-wrapper #contactDetails #govID span {
     margin-right: 1%;
 }
+
+#crm-container .crm-contact-page {
+    position: relative;
+    padding: 4px;
+}
+#crm-container #mainTabContainer {
+    background-color: transparent;
+    padding: 0px;
+}
+
+#mainTabContainer .ui-tabs-nav {
+    width: 15%;
+    float:left;
+    padding:4px;
+}
+#mainTabContainer li.crm-tab-button {
+    display: block;
```

Comment:
We are switching to a 2-space indentation to be consistent with CiviCRM standard (please use the [.editorconfig](https://github.com/civicrm/civihr/blob/develop/.editorconfig) file in your IDE)